plugin/ipam/cni.go

```diff
@@ -37,7 +38,15 @@ func (i *Ipam) Allocate(args *skel.CmdArgs) (types.Result, error) {
 	}
 	var ipnet *net.IPNet
 
-	if conf.Subnet == "" {
+	if len(conf.IPs) == 1 {
```

Comment:
What happens when 2 or more IPs are supplied?

Reply:
I am not sure how we can assign multiple IP addresses to the same interface we pass as part of `RuntimeConf`, when calling `AddNetworkList`.